definitions/infra-squid_cachemgr/definition.yml

```diff
@@ -3,15 +3,19 @@ type: SQUID_CACHEMGR
 
 synthesis:
   rules:
-    - identifier: squid_server
-      name: squid_server
```

Comment:
old entities that where created with this attribute will still be there until the TTL (24h) is reached

Just making sure you are aware and ok with this.